library/ssl_tls13_client.c

```diff
@@ -2046,9 +2045,7 @@ static int ssl_parse_server_psk_identity_ext( mbedtls_ssl_context *ssl,
     int ret = 0;
     size_t selected_identity;
 
-    if( ssl->conf->f_psk == NULL &&
```

Comment:
I got rid of that condition because the PSK callback is not used on the client side

Reply:
Correct :+1: